config/jobs/cert-manager/cert-manager-periodics.yaml

```diff
@@ -356,3 +356,52 @@ periodics:
       options:
       - name: ndots
         value: "1"
+
+- name: aws-tests
+  interval: 6h
+  agent: kubernetes
+  decorate: true
+  # extra_refs:
+  # - org: jetstack
+  #   repo: cert-manager
+  #   base_ref: master
+  annotations:
+    testgrid-create-test-group: 'true'
+    testgrid-dashboards: jetstack-cert-manager-master
+    testgrid-alert-email: cert-manager-dev-alerts@googlegroups.com
+    description: Runs the end-to-end test suite against a EKS cluster
+  # labels:
+  #   preset-service-account: "true"
+  #   preset-dind-enabled: "true"
+  #   preset-bazel-remote-cache-enabled: "true"
+  #   preset-bazel-scratch-dir: "true"
+  #   preset-cloudflare-credentials: "true"
+  #   preset-venafi-tpp-credentials: "true"
+  #   preset-venafi-cloud-credentials: "true"
+  spec:
+    containers:
+    - image: eu.gcr.io/jetstack-build-infra-images/bazelbuild:20210323-ad5071a-3.7.2
+      args:
+      - -c
+      - |
+        set -euo; \
+        git clone https://github.com/jetstack/cert-manager.git; \
+        git clone https://github.com/cert-manager/test-infra.git; \
+        cd test-infra/aws; \
+        terraform init; \
+        terraform apply -auto-approve; \
+        cd..; \
+        cd..; \
+        cd cert-manager; \
+        ./devel/run-e2e.sh --ginkgo.focus "Public ACME Server HTTP01 Issuer" --acme-server-url=https://acme-staging-v02.api.letsencrypt.org/directory --ingress-controller-domain=aws.e2e-tests.cert-manager.io --testing-acme-email=arshsharma461+acme-tests-1@gmail.com --kubernetes-config=/home/arsh/projects/forks/test-infra-1/aws/kubeconfig_cert-manager-cluster;
+      resources:
+        requests:
+          cpu: 3500m
+          memory: 12Gi
+      env:
+      - name: AWS_ACCESS_KEY_ID
+        value: "???"
+      - name: AWS_SECRET_ACCESS_KEY
+        value: "???"
```

Comment:
We can't have the access keys on GitHub so how will this work?

Reply:
It'll probably need to be a `Secret` in the cluster where the Prow job runs. The Prow job can then be configured to consume it via a preset like we do it with i.e Venafi creds- preset defined [here](https://github.com/jetstack/testing/blob/master/config/jobs/cert-manager/config.yaml#L22) and consumed added to the job [here](https://github.com/jetstack/testing/blob/master/config/jobs/cert-manager/cert-manager-periodics.yaml#L86). (The actual `Secret` would need to be created in the cluster separately).